src/Microsoft.ML.Transforms/NormalizerCatalog.cs

```diff
@@ -71,15 +71,15 @@ public static class NormalizationCatalog
         /// </format>
         /// </example>
         public static LpNormalizingEstimator LpNormalize(this TransformsCatalog catalog, string outputColumnName, string inputColumnName = null,
```

Comment:
This name isn't the best....

Reply:
How about NormalizeLpNorm?

---
In reply to: [264869544](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264869544) [](ancestors = 264869544)